hw/ip/pwrmgr/dv/sva/pwrmgr_rstmgr_if.core

```diff
@@ -0,0 +1,18 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:pwrmgr_rstmgr_sva:0.1"
```

Comment:
nit Ideally prefer to have the same fusesoc core name (AND the `.core` filename) as the entity it is representing. Recommend this:
pwrmgr_rstmgr_sva_if.core (which would also be the name on line 5)
pwrmgr_rstmgr_sva_if.sv

Reply:
I like this name much better, and seems like a good convention, so I applied it to the other sva_if files here.